examples/src/main/java/io/grpc/examples/experimental/CompressingHelloWorldClient.java

```diff
@@ -16,6 +16,8 @@
 
 package io.grpc.examples.experimental;
 
+import io.grpc.CompressorRegistry;
```

Comment:
I don't want any example that uses these. It seems like it is unnecessary, as well, since these are the defaults.

Reply:
Removed the unnecessary configuration as the review comment